README.md

```diff
@@ -196,16 +148,11 @@ Example:
 
 ```
  -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
+👤 **Wasyihun Sema**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
```

Comment:
- [ ] Kindly add your full social media links based on the way it's structured in the Author section. This helps you maintain professionalism in your work 👍